Source/Node/include/MFRC522Reader.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef MFRC522READER_HPP
```

Comment:
Do these defines need to be public?

Reply:
no, removed the public defines and system is still functional